packages/flutter/lib/src/foundation/tree_diagnostics_mixin.dart

```diff
@@ -569,12 +569,12 @@ class _NoDefaultValue {
   const _NoDefaultValue();
 }
 
-/// Marker object indicating that a DiagnosticNode has no default value.
```

Comment:
cc @jacob314 there are a number of changes relating to your recent patch throughout this PR, starting here